Let test succeed also on 32-bit builds, when sizeof(int) == sizeof(ssize_t)
--- nghttp2-master/tests/nghttp2_session_test.c.orig	2013-10-12 10:02:37.000000000 +0200
+++ nghttp2-master/tests/nghttp2_session_test.c	2013-10-16 20:24:25.343834489 +0200
@@ -3490,12 +3490,12 @@
              NGHTTP2_OPT_PEER_MAX_CONCURRENT_STREAMS,
              &sszval, sizeof(sszval)));
 
-  intval = 100;
+  charval = 100;
   CU_ASSERT(NGHTTP2_ERR_INVALID_ARGUMENT ==
             nghttp2_session_set_option
             (session,
              NGHTTP2_OPT_PEER_MAX_CONCURRENT_STREAMS,
-             &intval, sizeof(intval)));
+             &charval, sizeof(charval)));
 
   nghttp2_session_del(session);
 }
